tree/dataframe/test/dataframe_interface.cxx

```diff
@@ -371,9 +388,12 @@ TEST(RDataFrameInterface, GetNSlots)
 TEST(RDataFrameInterface, DefineAliasedColumn)
 {
    ROOT::RDataFrame rdf(1);
-   auto r0 = rdf.Define("myVar", [](){return 1;});
+   auto r0 = rdf.Define("myVar", [] { return 1; });
    auto r1 = r0.Alias("newVar", "myVar");
-   EXPECT_ANY_THROW(r0.Define("newVar", [](int i){return i;}, {"myVar"})) << "No exception thrown when defining a column with a name which is already an alias.";
+   auto mdefine = r0.Define("newVar", [] { return 42; }).Max<int>("newVar");
+   EXPECT_EQ(*mdefine, 42);
+   auto malias = r1.Max<int>("newVar");
```

Comment:
```suggestion
   auto malias = r1.Max<int>("newVar");
   EXPECT_EQ(*mdefine, 42);
```
for good style (run the event loop only once)?

Reply:
haha yes! :smile: